javaslang/generator/Generator.scala

```diff
@@ -1252,6 +1253,13 @@ def generateMainClasses(): Unit = {
                   return INSTANCE;
               }
             """)}
+
+            ${(i > 1).gen(xs"""
```

Comment:
@ronanM thank you for committing the changes.

Please also generate the `sequence` method for Tuple1 (see above) so that I can merge your PR.

Thank you for this feature!